experimental/packages/opentelemetry-api-metrics/src/NoopMeter.ts

```diff
@@ -120,14 +119,7 @@ export class NoopHistogramMetric extends NoopMetric implements Histogram {
   record(_value: number, _labels: Labels): void {}
 }
 
-export class NoopObservableBaseMetric extends NoopMetric implements ObservableBase {
-  observation(): Observation {
-    return {
-      observable: this as ObservableBase,
-      value: 0,
-    };
-  }
-}
+export class NoopObservableBaseMetric extends NoopMetric implements ObservableBase {}
```

Comment:
Does NoopMetric serve any purpose here or anywhere?

Reply:
Hm, the only use case can be `observable instanceof NoopMetric`. However, there are no API spec constraints on class hierarchy here, maybe we can remove it.